src/main/java/ee/tuleva/onboarding/comparisons/FundComparisonController.java

```diff
@@ -25,10 +24,18 @@
 
   private final FundComparisonCalculatorService fundComparisonCalculatorService;
 
-  @ApiOperation(value = "Compare the current user's return to estonian and world average")
+  private static Instant parseInstant(String format) {
```

Comment:
According to Clean Code, the order of methods in a class should be so that high-level details come at the top of the class and as you scroll down you get to more lower-level details. And methods are ordered so they are close to the ones that call them. The caller first, callee second. You can read more about this philosophy from this summary: https://softwareengineering.stackexchange.com/a/199317